app/api/domains/osu.py

```diff
@@ -518,11 +486,8 @@ async def osuSearchHandler(
         if bmap["ChildrenBeatmaps"] is None:
             continue
 
-        if USING_CHIMU:
-            bmap["HasVideo"] = int(bmap["HasVideo"])
-        else:
-            # cheesegull doesn't support vids
-            bmap["HasVideo"] = "0"
+        # some mirrors use a true/false instead of 0 or 1
+        bmap["HasVideo"] = int(bmap["HasVideo"])
```

Comment:
Instead of checking whether it's chimu, do this for every mirror as chimu is not the only mirror providing the HasVideo property as a true/false value. Kitsu for example does that too